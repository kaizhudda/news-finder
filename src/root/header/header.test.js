import React from 'react';
import { mount } from 'enzyme';
import Header from './header.ui';

let wrapped = mount(<Header />);

describe('Header', () => {
  it('should render the Header Component', () => {
    expect(wrapped).toMatchSnapshot();
  });
});