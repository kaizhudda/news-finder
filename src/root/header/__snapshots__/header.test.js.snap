// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the Header Component 1`] = `
<Header>
  <div
    className="Header row"
  >
    <div
      className="col-xs-12 col-lg-8 col-lg-offset-2"
    >
      <div
        className="header-content"
      >
        <h1>
          HN.
        </h1>
        <SearchBar
          onChange={[Function]}
          value=""
        >
          <form
            className="search-container"
          >
            <input
              id="search-bar"
              onChange={[Function]}
              placeholder="Search stories by title"
              type="text"
              value=""
            />
          </form>
        </SearchBar>
        <Toggle>
          <div
            className="dark-mode-toggle"
          >
            <button
              onClick={[Function]}
              type="button"
            >
              ☀
            </button>
            <span
              className="toggle-control"
            >
              <input
                checked={false}
                className="dmcheck"
                id="dmcheck"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                htmlFor="dmcheck"
              />
            </span>
            <button
              onClick={[Function]}
              type="button"
            >
              ☾
            </button>
          </div>
        </Toggle>
      </div>
    </div>
  </div>
</Header>
`;
